src/settings.js

```diff
@@ -721,6 +721,9 @@ var BINARYEN_ASYNC_COMPILATION = 1; // Whether to compile the wasm asynchronousl
 var BINARYEN_ROOT = ""; // Directory where we can find Binaryen. Will be automatically set for you,
                         // but you can set it to override if you are a Binaryen developer.
 
+var JS_FFI = 1; // Whether to legalize the JS FFI interfaces (imports/exports) by wrapping
+                // them to automatically demote i64 to i32 and promote f32 to f64
```

Comment:
Please add something like `. This is necessary in order to interface with JavaScript, both for asm.js and wasm.`